### PR TITLE
lock fix

### PR DESCRIPTION
**Pull request by StephenCoady**
## Motivation

fix the package-lock.json file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,8 @@
     "@ionic/storage": "2.1.3",
     "cordova-android": "^7.1.0",
     "cordova-ios": "4.5.4",
+    "cordova-plugin-aerogear-metrics": "@aerogear/cordova-plugin-aerogear-metrics@~0.3.0",
```
Comment (aidenkeating) [inline package.json]: I'm seeing a weird issue with these additions

```
npm ERR! Could not install from "@aerogear/cordova-plugin-aerogear-metrics@~0.3.0" as it does not contain a package.json file.
```

If I remove those, everything seems to work fine. Looks like the package-lock.json contained some local stuff that we don't want included.

**Comment by wtrocki** [comment]
WOW. I did not even know we have package.json lock. From my experience with Cordova it doesn't work really great. My vote will be to remove it. 
I tried to replicate that issue but it was resolved after I removed plugins folder.

**Comment by aidenkeating** [comment]
Happy for it to be removed too if it doesn't work great with the Cordova use-case

**Comment by wtrocki** [comment]
Yeah. We will jus remove it from repository and gitignore it.

**Comment by wtrocki** [comment]
@StephenCoady Can we remove lock?

**Comment by StephenCoady** [comment]
@wtrocki it was gitignored already so looks like it just wasn't git removed. done now

**Pull request merged**